octobot_evaluators/evaluators/evaluator_factory.py

```diff
@@ -66,6 +66,17 @@ async def create_evaluators(evaluator_parent_class,
     ]
 
 
+def create_temporary_evaluator_with_local_config(
+        evaluator_class,
+        tentacles_setup_config: tm_configuration.TentaclesSetupConfiguration,
+        specific_config: dict,
+        should_trigger_post_init=False):
+    evaluator_instance = evaluator_class(tentacles_setup_config, should_trigger_post_init=should_trigger_post_init)
+    evaluator_instance.logger = logging.get_logger(evaluator_instance.get_name())
+    evaluator_instance.specific_config = specific_config
+    return evaluator_instance
```

Comment:
I don't think it's a good idea to duplicate Evaluator class instantiation. I think we need to find a way to use the same code as `create_evaluator`